tests/unit/helpers/test_auth_flow_managers.py

```diff
@@ -52,3 +56,23 @@ def mock_b64encode(b: bytes):
 
     assert len(b64vals) == 2
     assert b64vals == [b"xyz", hashlib.sha256(b"abc123").digest()]
+
+
+def test_get_authorize_url_for_authorization_code():
+    mock_client = mock.Mock()
+    mock_client.client_id = "MOCK_CLIENT_ID"
+    mock_client.base_url = "https://auth.globus.org/"
+    flow_manager = GlobusAuthorizationCodeFlowManager(
+        mock_client, redirect_uri="https://foo.example.org/authenticate"
+    )
+
+    silly_string = "ANANAS_IS_PINEAPPLE_BUT_BANANE_IS_BANANA"
+
+    authorize_url = flow_manager.get_authorize_url()
+    assert authorize_url.startswith("https://auth.globus.org")
+    assert silly_string not in authorize_url
```

Comment:
I'm confused what the purpose of this assertion is?

Reply:
It's not terribly important; just a little habit or quirk of how I write many of my tests.
If I want to assert that calling some callable `foo` with a parameter `bar` causes a desired behavior, I like to assert the converse -- that calling `foo` without `bar` does not exhibit that behavior.

There are a lot of cases where this is sensible and makes the test better. And then there are cases like this one, in which it's hard to imagine the assertion ever failing.

Reply:
I see! Makes sense. Just making sure I'm not missing anything.